core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -1949,8 +1949,10 @@ public void showEmote(@NonNull GeyserPlayerEntity emoter, @NonNull String emoteI
         }
 
         EmotePacket packet = new EmotePacket();
-        packet.setEmoteId(emoteId);
         packet.setRuntimeEntityId(entity.getGeyserId());
+        packet.setXuid(""); // not required but cannot be null
```

Comment:
Comment is weird here since Protocol in general works like this and it's not specific to the field... ultimately though not a dealbreaker.

Reply:
oke